src/components/common/TmBalance.vue

```diff
@@ -1,230 +1,104 @@
-                        ).concat(` ` + preferredCurrency)
-                      }}</span
-                    >
-                  </div>
-                </div>
-
-                <div
-                  v-if="
-                    !isMultiDenomNetwork &&
-                      overview.totalRewards &&
-                      overview.totalRewards > 0.001
-                  "
-                  class="rewards"
-                >
-                  <h3>Total Rewards</h3>
-                  <h2>
-                    +{{
-                      overview.totalRewards
-                        | bigFigureOrShortDecimals
-                        | noBlanks
-                    }}
-                  </h2>
-                </div>
-              </div>
-
-              <div v-if="isMultiDenomNetwork" class="row values-container">
-                <div
-                  v-for="balance in filteredMultiDenomBalances"
-                  :key="balance.denom"
-                  class="currency-div"
-                >
-                  <div class="available-atoms">
-                    <h3>
-                      {{ balance.denom }}
-                    </h3>
-                    <h2>
-                      {{ balance.amount | bigFigureOrShortDecimals }}
-                    </h2>
-                  </div>
-                  <div class="rewards multi-denom">
-                    <h2
-                      v-if="calculateTotalRewardsDenom(balance.denom) > 0.001"
-                    >
-                      +{{
-                        calculateTotalRewardsDenom(balance.denom)
-                          | bigFigureOrShortDecimals
-                      }}
-                    </h2>
-                  </div>
-                  <div
-                    v-if="
-                      balance.fiatValue &&
-                        balance.fiatValue.amount > 0 &&
-                        preferredCurrency
-                    "
-                    class="total-fiat-value fiat-value-box"
-                  >
-                    <span>{{
-                      bigFigureOrShortDecimals(balance.fiatValue.amount).concat(
-                        ` ` + preferredCurrency
-                      )
-                    }}</span>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
+    ></div>
+
+    <div v-else class="overview">
+      <div class="overview-row titles">
+        <h3 class="title cell total">Total</h3>
+        <h2 class="title cell">Rewards</h2>
+        <h2 class="title cell">Available</h2>
+        <h2 class="title cell"></h2>
       </div>
-      <div class="button-container">
-        <TmBtn
-          class="send-button"
-          value="Send"
-          type="secondary"
-          @click.native="onSend()"
+
+      <ul v-if="overview && !overview.totalStakeFiatValue">
+        <BalanceRow
+          :crypto-amount="overview.totalStake"
```

Comment:
i see, this should be queried in the balance query

Reply:
`total` can be queried through balances but `available` cannot

Reply:
also, would be nice if `rewards` were included in the balances query

Reply:
sure, but isn't that just putting everything into one query? better to separate the concerns don't you think?